libs/website.js

```diff
@@ -94,13 +94,13 @@ module.exports = function(logger){
     };
 
 
-    //If an html file was changed reload it
-    watch('website', function(filename){
+    // if an html file was changed reload it
+    /* requires node-watch 0.5.0 or newer */
+    watch(['./website', './website/pages'], function(evt, filename){
         var basename = path.basename(filename);
         if (basename in pageFiles){
-            console.log(filename);
             readPageFiles([basename]);
-            logger.debug(logSystem, 'Server', 'Reloaded file ' + basename);
+            logger.special(logSystem, 'Server', 'Reloaded file ' + basename);
```

Comment:
why it changed?

Reply:
This is only update for latest nodejs, some javascript coding problem.